qutip/tests/test_wigner.py

```diff
@@ -94,16 +94,19 @@ def test_wigner_fock():
         # calculate the wigner function using qutip and analytic formula
         W_qutip = wigner(psi, xvec, yvec, g=2)
```

Comment:
Currently the default method is 'clenshaw'. Do we want to specify 'iterative' method here or remove the line completely?